ui/src/views/Documentation/Reference/SchemaIndex.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import sortBy from 'lodash/sortBy';
+import filter from 'lodash/filter';
+import Entry from './Entry';
+import HeaderWithAnchor from '../components/HeaderWithAnchor';
+import references from '../../../../../generated/references.json';
+
+@withRouter
+export default class SchemaIndex extends Component {
+  // Filters only schemas, sorted by $id
+  getSortedReferences = () => {
+    const sortedReferences = filter(references, entry =>
+      entry.filename.startsWith('schemas/')
+    );
+
+    return sortBy(sortedReferences, ['content.$id']);
+  };
+
+  render() {
+    const sortedReferences = this.getSortedReferences();
+
+    return (
+      <div>
+        <HeaderWithAnchor>Schema Index</HeaderWithAnchor>
+        <Fragment>
+          {sortedReferences.map(entry => (
+            <Entry
+              key={entry.content.$id}
+              serviceName="unknown"
```

Comment:
Instead of passing `unknown`, we could make `serviceName` not required. You should be able to do that in `static PropTypes`.

Reply:
For the moment, I want this to show up if it's used to generate a URL (it shouldn't be..).  There's a bug to fix this that's assigned to me.

Reply:
@helfi92 @djmitche So I should keep it as `unknown` ?

Reply:
Yes -- I'll change it in #657.